test/inline_module_test.cpp

```diff
@@ -0,0 +1,569 @@
+#include <migraphx/dead_code_elimination.hpp>
+#include <migraphx/inline_module.hpp>
+#include <migraphx/pass_manager.hpp>
+#include <migraphx/instruction.hpp>
+#include <basic_ops.hpp>
+#include <migraphx/operators.hpp>
+#include <migraphx/make_op.hpp>
+
+#include <test.hpp>
+
+void run_pass(migraphx::program& p)
+{
+    auto* mm = p.get_main_module();
+    migraphx::run_passes(*mm, {migraphx::inline_module{}, migraphx::dead_code_elimination{}});
+    migraphx::run_passes(p, {migraphx::dead_code_elimination{}});
```

Comment:
You need to add `migraphx::inline_module` here.

Reply:
fixed